include/boost/geometry/algorithms/discrete_frechet_distance.hpp

```diff
@@ -153,19 +153,35 @@ inline typename distance_result
             typename point_type<Geometry2>::type,
             Strategy
         >::type
-frechet_distance(Geometry1 const& g1, Geometry2 const& g2, Strategy const& strategy)
+frechet_distance(Geometry1 const& geometry1, Geometry2 const& geometry2, Strategy const& strategy)
```

Comment:
Here, the description is missing. Also the functions should be called the same as the file so `discrete_frechet_distance()`.